### ISSUE TITLE
Formatting continues to next line in document editor

### ISSUE THREAD

**Issue by CameronDevine** [TheoTech/spf.io#7]
In the document editor when a multiline bold section of text is pasted into the rich text mode of the document editor the following markdown is created,

```markdown
**How could this be the Messiah?
He grew up here.
He’s one of us.
We know him - we know his family.
Who does he think he is?**
```

As each line appears to be parsed individually, this would not make all the lines bold. Therefore this should be,

```markdown
**How could this be the Messiah?**
**He grew up here.**
**He’s one of us.**
**We know him - we know his family.**
**Who does he think he is?**
```

**Comment by chrislim** [comment]
Would you like it to remain combined on one line? 
Or would you prefer for it to still be broken out and retain its formatting?

**Comment by CameronDevine** [comment]
Either way would be fine with me. Whatever way you choose to fix it, I think the main goal should be to make the rich text view of the document editor match the formatting in the audience view.

It may also be possible to fix this issue and #5 at the same time.

**Comment by chrislim** [comment]
Fixed in latest release.
